doc/content_item_fields.md

```diff
@@ -115,8 +115,8 @@ for the Latin America and Caribbean region").
 ISO 8601 formatted timestamp. Present in all contexts.
 
 This is the time at which the content was last publicly visibly updated. This
-can be used for sorting by update date, but will not be changed for minor
-updates.
+is used for sorting documents by update date. It should change when there is a
+major update and should not change for a minor update.
```

Comment:
It might be better to word this differently.  Something like "This is the update date that should be surfaced to the user".  It's up to the publishing app when that should be.  It will typically coincide with a major update, but doesn't have to.